dashboard/app/models/channel_token.rb

```diff
@@ -87,14 +87,12 @@ def self.find_channel_token(level, user_storage_id, script_id)
   def self.create_channel(ip, project, data: {}, src: nil, type: nil, remix_parent_id: nil, standalone: true, level: nil)
     if src
       data = project.get(src)
-      data['name'] = "Remix: #{data['name']}"
-      data['hidden'] = false
-      data['frozen'] = false
+      data.merge!(name: "Remix: #{data['name']}", hidden: false, frozen: false)
     end
 
     timestamp = Time.now
     project.create(
-      data.merge('createdAt' => timestamp, 'updatedAt' => timestamp),
+      data.merge(createdAt: timestamp, updatedAt: timestamp),
```

Comment:
This change allows the `createdAt' and `updatedAt` values to be updated with `Time.now` if project is remixed (so that the values are not those from the original/src project).
Before this change, both the string and symbole keys existed in `data` and the symbol keys have precedence